plugins/rikki/heroeslounge/components/updatematch/default.htm

```diff
@@ -84,6 +87,12 @@ <h3 class="card-header text-center" style="background-color: #fff">
                 </div>
             </div>
             {% endfor %}
+
+            <div>
+                <button class="btn btn-primary btn-block" type="button" data-toggle="modal" data-target="#reschedule_{{__SELF__.match.id}}" title="Reschedule match">
+                    Reschedule
+                </button>
+            </div>
```

Comment:
I think this looks better above the Game stuff (but I have not actually taken a look at it, so I will leave it at your discretion)